### ISSUE TITLE
Bug in Run-AlPipeline code

### ISSUE THREAD

**Issue by LukasK7** [microsoft/navcontainerhelper#1371]
**Describe the issue**
Bug in Run-AIPipeline.ps1 file.
```
"additionalParameters" = @("--volume $($baseFolder):c:\sources")
```
It should be like this
```
"additionalParameters" = @("--volume ${baseFolder}:c:\sources")
```

**Screenshots**

![image](https://user-images.githubusercontent.com/58076001/96118229-dea30700-0ef3-11eb-86ab-91659cca047b.png)

**Comment by ernestasjuska** [comment]
To correct my colleague. Bug is that base folder can contain spaces and it is expanded incorrectly, so docker run receives
--volume Path With Space:c:\sources instead --volume "Path With Space:c:\sources".

**Comment by freddydk** [comment]
I guess it should be:
```
"additionalParameters" = @("--volume ""$($baseFolder):c:\sources""")
```

**Comment by freddydk** [comment]
Fix shipped in 1.0.10
